test/integration/test_focal.py

```diff
@@ -19,7 +19,7 @@ class TestIntegrationFocalLoss:
     num_classes = 2
 
     # focal loss
-    alpha = 2.0
+    alpha = 0.5
```

Comment:
old `alpha` is actually not working. To get the same result as before, `alpha` should be balanced, that is, `0.5`.